### ISSUE TITLE
Fluent Routes bug

### ISSUE THREAD

**Issue by spaantje** [laravel-shift/workbench-issues#48]
**Desktop:**
 - OS: Mac
 - OS Version: Big Sur, 11.5.2
 - App Version: 0.6.3

**Describe the bug**
I have run the Fluent Routes task but it is not translating the route correctly. A image of the diff:
![image](https://user-images.githubusercontent.com/477362/138452717-5cbda362-807b-4651-9b83-7c2f73f5c6e9.png)

**To Reproduce**

This:
```php
Route::prefix('{product}')->group(function () {
    Route::resource('contact', ContactController::class, ['only' => ['store', 'index', 'show']]);
    Route::resource('nieuws', ProductNewsController::class, ['only' => ['index', 'show'], 'parameters' => ['nieuws' => 'news']]);
});
```

Gets translated to: 
```php
Route::prefix('{product}')->group(function () {
    Route::resource('contact', ContactController::class)->only('store', 'index', 'show');
    Route::resource('nieuws', ProductNewsController::class, [ 'parameters' => ['nieuws' => 'news']]);
});
```

But should be: 
```php
Route::prefix('{product}')->group(function () {
    Route::resource('contact', ContactController::class)->only('store', 'index', 'show');
    Route::resource('nieuws', ProductNewsController::class)->only('index', 'show')->parameters(['nieuws' => 'news']);
});
```
(So in short; it removed the only and did not move the parameters.)

**Comment by jasonmccreary** [comment]
Thanks for reporting. We'll get this reviewed and patched next week.

**Comment by spaantje** [comment]
Thanks. No rush :-)

**Comment by jasonmccreary** [comment]
@spaantje, this should have been fixed in version 0.7.5. Let us know if not.
